app/services/utils/raster.py

```diff
@@ -0,0 +1,51 @@
+from rio_tiler.io import Reader
+from rasterio import features
+from geopandas import GeoDataFrame
+from shapely import geometry
+
+
+# TODO: become generic in order to be able to reuse
+def crop_raster(raster_path, polygon):
+    with Reader(raster_path) as image:
+        img = image.feature(polygon)
+
+    return img.render(
+        add_mask=True,
+        colormap={
+            0: (255, 0, 0, 255),
+            1: (128, 204, 102, 255),
+            2: (232, 214, 107, 255),
+        },
+    )
+
+
+def get_raster_values(raster_path, polygon):
+    crs = "EPSG:9377"
```

Comment:
where does this value come from?

Reply:
It's the plane coordinate projection that Julián recommended for areas in Colombia. [EPSG:9377
](https://epsg.io/9377)